test/unit/test_runner.py

```diff
@@ -93,16 +101,13 @@ def test__execute_model__view(self,
                                   mock_adapter_rename,
                                   mock_adapter_execute_model):
         model = self.model.copy()
-
-        dbt.runner.execute_model(
-            self.profile,
-            model,
-            existing=self.existing)
+        runner = self.get_model_runner(model)
+        runner.execute(model, existing=self.existing)
 
         dbt.adapters.postgres.PostgresAdapter.drop.assert_not_called()
 
         mock_adapter_truncate.assert_not_called()
-        mock_adapter_rename.assert_not_called()
+        mock_adapter_rename.assert_called_once()
```

Comment:
this makes me nervous. was this not working properly before? my understanding that view models would just be created with the final name if they did not previously exist (vs created and then renamed)